#! /bin/sh /usr/share/dpatch/dpatch-run
## 55_touch_pot.dpatch by  <jrnieder@gmail.com>
##
## DP: Bump POT-Creation-Date to ensure gmo files are rebuilt.
## http://bugs.debian.org/671807

@DPATCH@
diff --git a/po/coreutils.pot b/po/coreutils.pot
index 38e9f112..1e60a430 100644
--- a/po/coreutils.pot
+++ b/po/coreutils.pot
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: GNU coreutils 8.13\n"
 "Report-Msgid-Bugs-To: bug-coreutils@gnu.org\n"
-"POT-Creation-Date: 2011-09-08 17:09+0200\n"
+"POT-Creation-Date: 2012-09-10 20:15-0700\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
